README.rst

```diff
@@ -13,6 +13,8 @@ Features
 
   * Check redirect locations
 
+  * Check HTTP header values
```

Comment:
Should we clarify that Smoketest is checking HTTP _response_ header values? Or maybe that's implied.

Would it make sense to combine this bullet with the one directly above? "Check HTTP response header values, including `Location`"?

Reply:
I changed the language to use the word "response" in 3f3edac.

I left the line separate from the redirect line. My thinking is that the two kinds of tests are specified differently in expectations files, and redirect tests can be more complicated due to following redirects and mangling the location. Keeping the two lines separate at least hints at the fact that the code treats them as two different things.